calico-enterprise_versioned_docs/version-3.16/_includes/release-notes/_v3.16.3-release-notes.mdx

```diff
@@ -0,0 +1,16 @@
+xx June 2023
+
+:::note
+Release 3.16.2 was incorrectly tagged during our release process so 3.16.3 is the next patch release after 3.16.1. (There is no 3.16.2 patch release).
+:::
+
+### Egress gateways
+
+The Tigera operator automatically detects and configures egress gateways to use the same iptables backend as calico-node.
+
+### Known issues {#known-issues-v3.16.3}
+
+- Enabling tcp stats might result in a memory leak because of a kernel bug.
+- The ClusterInformations resource is showing the wrong versions for `calicoVersion` and `cnxVersion`.
```

Comment:
I don't think so? Is this carried over from v3.16.1?

Reply:
Yes, I think it now reports the right versions.

Reply:
Removed.